### PR TITLE
chore: update nodejs to v22.1.0

### PR DESCRIPTION
**Pull request by stefreak**
- chore: update nodejs to v22.1.0
- chore: remove UV_USE_IO_URING=0 fix as libuv 1.48 landed
- chore: update nodejs in circleci
- chore(images): update circleci-runner to use NodeJS 22.1.0

### CODE REVIEW

File: garden-sea/src/node.rs [hunk 1]
```diff
@@ -39,15 +39,17 @@ where
...
-    command.env(
-        "UV_USE_IO_URING",
-        env::var("GARDEN_SEA_UV_USE_IO_URING").unwrap_or("0".into()),
-    );
+    // Experimental compile cache feature. Defaults to false for now, but we can enable it by default later if it works well.
+    // See also https://nodejs.org/api/cli.html#node_compile_cachedir
+    let enable_compile_cache = env::var("GARDEN_COMPILE_CACHE").unwrap_or("false".into());
+    if enable_compile_cache == "true" || enable_compile_cache == "1" {
+        let cache_dir = path.join("v8cache");
+        fs::create_dir(cache_dir.clone())?;
```
Comment (stefreak) [inline garden-sea/src/node.rs:47]: we need to use `create_dir_all` I think otherwise it'll fail if it already exists

**Comment by vvagaytsev** [comment]
We also need to update `circleci-runner` image.

**Comment by vvagaytsev** [comment]
> We also need to update `circleci-runner` image.

I updated it :)
* 70e28861e7f95b2cf0df847749fbdbc585b906f8
* 223bdff67e33a65a9164d592d21890552b89f862

**Review by vvagaytsev — DISMISSED** [review]
🎉 👍

**Review by vvagaytsev — DISMISSED** [review]

**Review by vvagaytsev — APPROVED** [review]

**Review by vvagaytsev — APPROVED** [review]

**Review by vvagaytsev — APPROVED** [review]

**Pull request merged**